homeassistant/components/sensor/sonarr.py

```diff
@@ -166,6 +181,8 @@ def unit_of_measurement(self):
     def device_state_attributes(self):
         """Return the state attributes of the sensor."""
         attributes = {}
+        if not self.available:
```

Comment:
Good to know, I had gotten an error but it was because i didn't initialize my data array, i removed this check and everything is good.